conftest.py

```diff
@@ -352,13 +347,11 @@ def test_shutdown():
 
 
 @pytest.fixture
-def run_manager(mocker, request_mocker, upsert_run, query_viewer):
+def run_manager(mocker, mock_server):
     """This fixture emulates the run_manager headless mode in a single process
     Just call run_manager.test_shutdown() to join the threads
     """
     with CliRunner().isolated_filesystem():
-        query_viewer(request_mocker)
-        upsert_run(request_mocker)
         run_manager = fake_run_manager(mocker)
         yield run_manager
         wandb.uninit()
```

Comment:
The mock server is way better than one-off mocks.  Ideally we annotate tests with markers to make the mock_server behave differently.  This change get's rid of all the no-mock-server for address errors, should have done it long ago.